### ISSUE TITLE
How to deal with other types

### ISSUE THREAD

**Issue by nicknikolov** [inokawa/remark-slate-transformer#37]
Whats the best way to add support for non-standard markdown types like https://github.com/landakram/remark-wiki-link or any other custom extensions to the mdAST.

Thanks!

**Comment by inokawa** [comment]
Hi, @nicknikolov 
unfortunatelly, this lib doesn't have a feature to handle new custom markdown type but I think it's worth to add option to handle them.

All of [mdast](https://github.com/syntax-tree/mdast)(ast in remark) nodes has `type` field, and this lib use it to detect kind of node.
For example remark-math(only one custom type supported in this lib) shows `Parses $ as inlineMath and $$ as math nodes` in https://github.com/remarkjs/remark-math#packages so I was able to support them.

However the node of remark-wiki-link looks like it doesn't have `type` field, so this lib could not detect its node even if I added the feature.
